catalog.html

```diff
@@ -4,11 +4,17 @@
 <head>
   <meta charset="utf-8">
   <title>HTML Academy: Техномарт</title>
+
+
+    <link rel="icon" href="favicon.ico">
+    <link rel="icon" href="img/favicon-small.svg" type="image/svg+xml">
+    <link rel="apple-touch-icon" href="img/180.png">
+    <link rel="manifest" href="manifest.webmanifest">
 </head>
 
 <body>
   <header>
-    <a href="index.html">Техномарт</a>
+      <a href="index.html"><img src="img/main-logo1.svg" width="220" height="47" alt="Главный логотив"></a>
```

Comment:
```suggestion
      <a href="index.html"><img src="img/main-logo.svg" width="220" height="47" alt="Логотип Техномарт"></a>
```
Лучше указать такой альт для картинки. Он более наглядно показывает какая именно картинка

Так же, если есть шанс самому указывать названия картинок, лучше их четко описывать, и не добавлять всякие нумерации. Чтобы из названия было понятно что именно за картинка и к чему относится